### PR TITLE
Hw01

### PR DESCRIPTION
**Pull request by lendijoy**

### CODE REVIEW

**Comment by lendijoy** [comment]
@roxannefelig 
@khoanguyen0812

File: hw01/hw01_exploring_data.Rmd [hunk 1]
```diff
@@ -1,20 +1,61 @@
...
+```
 
+### Clean it up
+It's probably not good practice to just throw out data, but let's do it anyway and keep only complete cases.
 ```{r}
-bfi$gender<- factor(bfi$gender, levels = c(1,2), labels = c("male", "female"))
-bfi$education<- factor(bfi$education, levels = c(1,2,3,4,5), labels = c("HS", "finished HS", "some college", "college grad", "grad degree"))
-```
\ No newline at end of file
+x <- na.omit(bfi)
```
Comment (roxannefelig) [inline hw01/hw01_exploring_data.Rmd:40]: Does this omit NA values, or whole cases with any NA responses?

File: hw01/hw01_exploring_data.md [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+    ##  1st Qu.:21.00  
+    ##  Median :26.00  
+    ##  Mean   :29.51  
+    ##  3rd Qu.:36.00  
+    ##  Max.   :86.00
+
+Lastly, let’s take a look at education level by gender.
+
+``` r
+table(x$gender, x$education)
```
Comment (roxannefelig) [inline hw01/hw01_exploring_data.md:162]: this might also be nice to see using the plot function

**Review by roxannefelig — COMMENTED** [review]
Your data exploration was really well-done, and there were several functions that you used that I'm making note of to incorporate more often!

File: hw01/hw01_exploring_data.Rmd (hunk 1 above)
Comment (lendijoy) [reply hw01/hw01_exploring_data.Rmd:40]: It omits all incomplete cases, keeping only those that are complete.

**Comment by khoanguyen0812** [comment]
Hi Lendi,

Your *__readme__* document for this repository is amazing! The one thing I would do is to maybe add some pictures for it to make it more vibrant.

I do not know much about R and I am quite new to it so I am unable to provide you with the higher-level useful information to your Rmd document. But from what I can see with my limited knowledge, your report is superb! Keep up the amazing work!

File: hw01/hw01_exploring_data.md (hunk 2 above)
Comment (lendijoy) [reply hw01/hw01_exploring_data.md:162]: Thanks for the suggestion! I wasn't sure what the best way to make a plot would be, but we learned some today 😄

**Comment by khoanguyen0812** [comment]
@bwiernik

**Pull request merged**